### PR TITLE
Added new layout 1 rows x 14 columns

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

**Comment by mkurek** [comment]
:ok:

File: src/ralph/assets/models/choices.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ class ModelVisualizationLayout(Choices):
     layout_4x2 = _('4 rows x 2 columns').extra(css_class='rows-4 cols-2')
     layout_2x4 = _('2 rows x 4 columns').extra(css_class='rows-2 cols-4')
     layout_2x8 = _('2 rows x 8 columns').extra(css_class='rows-2 cols-8')
+    layout_1x14 = _('1 rows x 14 columns').extra(css_class='rows-1 cols-14')
```
Comment (mkurek) [inline src/ralph/assets/models/choices.py]: move it to the end

**Comment by szok** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**